plugins/dice.js

```diff
@@ -3,8 +3,21 @@ const _ = require('lodash');
 module.exports = (function(){
 
 	let bot,
-		reDice = /([1-9][0-9]{0,2})(d|D)([1-9][0-9]{0,3})((\+|-|\*|\/)([0-9]{1,3}))?/g,
-		reMessage = /(^(\!roll )[1-9][0-9]{0,2})(d|D)([1-9][0-9]{0,3})((\+|-|\*|\/)([0-9]{1,3}))?/g;
+		reDice = /([1-9][0-9]{0,2})(d|D)([1-9][0-9]{0,3})((\+|-|\*|\/)([0-9]{1,3}))?( *(&[g|l]t;=?|=) *[0-9]*)?/g,
+		reMessage = /(^(!roll )[1-9][0-9]{0,2})(d|D)([1-9][0-9]{0,3})((\+|-|\*|\/)([0-9]{1,3}))?/g;
+
+	function checkTarget(result, quantifier, target) {
+		let text = ' `Failure!`',
+			quantifiers = {
+				'&gt;': (a,b) => a>b,
+				'&lt;': (a,b) => a<b,
+				'&gt;=': (a,b) => a>=b,
+				'&lt;=': (a,b) => a<=b,
+				'=': (a,b) => a===b
+			};
+		if (quantifiers[quantifier.toLowerCase()](parseFloat(result), parseFloat(target))) text = ' `Success!`';
+		return text;
+	}
```

Comment:
this is getting nitpicky, there really isnt anything wrong with what you have (except for maybe the parsefloat()) but here is how I would write this function:
```js
function checkTarget(result, quantifier, target) {

	//do your argument manipulation, defaulting, etc at the top of the function
	//making your use of them more clear
	quantifier = quantifier.toLowerCase();
	//either use lodash's parseFloat or use parseFload(result, 10)
	result = _.parseFloat(result);
	target = _.parseFloat(target);

	let quantifiers = {
		//personal style preferences
		'&gt;': (a, b) => a > b,
		'&lt;': (a, b) => a < b,
		'&gt;=': (a, b) => a >= b,
		'&lt;=': (a, b) => a <= b,
		'=': (a, b) => a === b
	};

	//this line is much easier to read with the manipulation stuff already done
	if (quantifiers[quantifier](result, target)){
		//I personally only use bracket-less if statements when doing an early bail-out return,
		//like when the function should just not run - errors or something like that
		return ' `Success!`';
	}
	//no need to store the result, just return it where necessary
	return ' `Failure!`'
}
```